docs/apis/avalanchego/apis/p-chain.md

```diff
@@ -908,23 +916,27 @@ curl -X POST --data '{
 
 ### platform.getCurrentSupply
 
-Returns an upper bound on the number of AVAX that exist. This is an upper bound because it does not account for burnt tokens, including transaction fees.
+Returns an upper bound on amount of tokens that exist that can stake the requested subnet. This is an upper bound because it does not account for burnt tokens, including transaction fees.
 
 #### **Signature**
 
 ```sh
-platform.getCurrentSupply() -> {supply: int}
+platform.getCurrentSupply({
+    subnetID: string // optional
+}) -> {supply: int}
 ```
 
-- `supply` is an upper bound on the number of AVAX that exist, denominated in nAVAX.
+- `supply` is an upper bound on the number of tokens that exist.
```

Comment:
```suggestion
- `supply` is an upper bound on the number of tokens that exist, denominated in 10^-9 of the native token of the Subnet, e.g., nAVAX on the Avalanche Primary Network. 
```

Reply:
The suggested comment isn't correct

Reply:
What is the correct denomination then?

Reply:
Depends on the Asset - it is not known by the P-chain.